### DISCUSSION TITLE
Oculus store version?

### DISCUSSION THREAD

**Discussion by Spudhead** [QuestCraftPlusPlus/QuestCraft#268] [Q&A]
I bought minecraft a long time ago from the oculus store, (big mistake). Is there anyway to make this work?

**Comment by ShinxDev** [comment]
Their working on bringing it to app lab

**Reply by wellsilver** [reply]
thats not what they are talking about

**Comment by wellsilver** [comment]
Minecraft on the oculus store is free so you might be mistaking it for Microsoft store

and no there isn't a way to make it work, however you can connect your quest to your computer and use minecraftVR that way

**Comment by Spudhead** [comment]
It’s free but you still have to pay for an account to use it and it’s different from both the Java and ms store versions. I’ve had this problem since I got it years ago. It’s SUPPOSED to be the ms store version but it won’t sign in if you use the account on the ms version. I tried it again yesterday and now I can’t play it at all.

On Apr 4, 2022, at 1:23 AM, wellsilver ***@***.***> wrote:

﻿

Minecraft on the oculus store is free so you might be mistaking it for Microsoft store

**Reply by wellsilver** [reply]
I dont get it, you need a payed account to use something free? Are you being ripped off?


also you replied via email??? absolute chad here

**Reply by ShinxDev** [reply]
Your not being ripped off, you have to OWN minecraft java to even install the game because they dont want microsoft to use them.

**Comment by Spudhead** [comment]
Maybe I’m missing something… to play minecraft you have to sign in with an account right? And you have to pay for that account. The quest craft app is free, but you still you still need a minecraft account (which is a paid account) to play. Or am I missing something?

I downloaded minecraft from the oculus App Store years ago, for free, but paid for an account to use it. This account is different from the ms store version and Java versions of minecraft.

On Apr 5, 2022, at 5:50 AM, wellsilver ***@***.***> wrote:

﻿

I dont get it, you need a payed account to use something free? Are you being ripped off?

also you replied via email??? absolute chad here

**Reply by wellsilver** [reply]
You arent missing anything,
the Oculus App Store is a addon for the bedrock version of minecraft to use with VR (minecrafts official VR) I dont know what you payed for, but there shouldnt be any kind of payment as its just a free addon to make it work

**Comment by veneficium42** [comment]
You need to buy minecraft java edition and log in with the account you used to buy minecraft java edition.
So no., if you bought minecraft from the oculus store you won't be able to use questcraft.

**Comment by Spudhead** [comment]
Thank you. That pretty much answers it.



On Apr 6, 2022, at 2:59 AM, FedFer98123 ***@***.***> wrote:

﻿

You need to buy minecraft java edition and log in with the account you used to buy minecraft java edition.
So no., if you bought minecraft from the oculus store you won't be able to use questcraft.

**Comment by Tman564ws1** [comment]
What it could be like is quest craft But in the description, it says you have to buy Java Minecraft. And sign in with your Microsoft account. I don't know if Oculus would allow this but it would be a great test. Because if they do, that means a lot more people will be able to do this.  Cuz let's say your brother owns Java but he Never lets you use his computer or he threw it away but he still has the Microsoft account. Well then there's a use for it.
